website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md

```diff
@@ -296,12 +308,12 @@ Developer accounts will be able to query the Proxy Server using SQL, but will no
   <div>
     <div>Metrics cannot be defined on <a href="https://docs.getdbt.com/docs/build/materializations#ephemeral">ephemeral models</a> because we reference the underlying table in the query that generates the metric so we need the table/view to exist in the database. If your table/view does not exist in your database, you'll likely see this error message:
```

Comment:
```suggestion
    <div>Metrics cannot be defined on <a href="https://docs.getdbt.com/docs/build/materializations#ephemeral">ephemeral models</a> because we reference the underlying table in the query that generates the metric so we need the table/view to exist in the database. If your table/view does not exist in your database, you might see this error message:
```